src/ray/gcs/gcs_server/default_job_info_access_handler.cc

```diff
@@ -0,0 +1,17 @@
+#include "default_job_info_access_handler.h"
+
+namespace ray {
+namespace rpc {
+void DefaultJobInfoAccessHandler::HandleAddJob(
```

Comment:
Shouldn't these be abstract methods?

Reply:
> Is this used only for testing?

Yes, It is just a demo of the frame of gcs server. By imitating this demo, we can implement more other interfaces (e.g. ActorJobInfoAccessHandler, NodeInfoAccessHandler ...)